c10/util/Optional.h

```diff
@@ -10,8 +10,6 @@
 
 namespace c10 {
 // NOLINTNEXTLINE(misc-unused-using-decls)
-using std::bad_optional_access;
```

Comment:
Why remove the aliasees here, it could break out of tree extensions for no reason.

Reply:
We have a series of patches to move to std::optional and we try not to break existing code as possible...

Reply:
Reverted